config/locales/service_providers/en.yml

```diff
@@ -1,6 +1,17 @@
 ---
 en:
   service_providers:
+    homes_mil:
+      account_page:
+        body: Your old HOMES.mil username and password won’t work. Please create a
+          login.gov account using the same email address you use for HOMES.mil.
+      body_html: Your old HOMES.mil username and password won’t work. Please %{link}
+        using the same email address you use for HOMES.mil.
+      create_account_link: create a login.gov account
+      create_account_page:
+        body: Please create a login.gov account using the same email address you use
+          for HOMES.mil
+      header: First time here from HOMES.mil?
```

Comment:
Looks like there's a lot of duplication here. The only difference in most cases is the name of the SP. Should we think about DRYing this up?

Reply:
Moncef we discussed this in a previous PR and that was precisely my concern: that we'd have duplicate entries for every SP.  I acquiesced.  Now some of the SPs have actually diverged wrt to the wording.  Moreover, some use the absence of a translation to drive the conditional display of help text on certain pages.  If we were going to refactor this I'd love to start with the code first and make it all data driven.  The sps hard coded in the source is the long pole in the tent.

Reply:
Sounds good. I was just making an observation, not proposing a specific solution. I couldn't remember the details of the last conversation.